README.md

```diff
@@ -76,14 +87,18 @@ pipeline step for mabl. This can be written by hand or created via the
 -   continueOnPlanFailure: Continue if your plan/tests fail
 -   environmentId: Selects the environment to run deployments against
     (one of environmentId or applicationId is required)
--   restApiKey: The apiKey of the desired deployment workspace -
+-   restApiKeyName: The name of the API key secret of the desired deployment workspace -
```

Comment:
Sorry to be a pain, but now that we have started using credential and it has ID `name of API key secret` should be probably be `ID of API key secret` (restApiKeyId?).